templates/assets/css/vlabs-style.css

```diff
@@ -63,13 +63,25 @@
 }
 
 .sidebar {
-  border-right: 2px dotted #89a7c4;
+  /* border-right: 2px dotted #89a7c4; */
+  border: thin solid #e8e8e8;
   font-size: 1.2rem;
   font-weight: bold;
-  overflow-x: hidden;
+  overflow: hidden;
   box-sizing: content-box;
 }
 
+@media (max-width: 991.9px){
```

Comment:
max-width 991 is sufficient, you don't need to use the decimal point.
Also try to see if the built in bootstrap breakpoint can be used instead of hard coding the breakpoints. Specifically, use the media-breakpoint-up and media-breakpoint-down mixins.
https://getbootstrap.com/docs/5.0/layout/breakpoints/

Reply:
I checked importing bootstrap mixins in our code. But since we are not using Sass, it is impossible to import those mixins.

Reply:
Moreover for floating point valued width, I found out that bootstrap follows the same logic to make it more accurate.
![image](https://user-images.githubusercontent.com/31105595/123192579-d6f32380-d4c0-11eb-86cf-c26e1b199b37.png)
Hence should I change it to 991 or 991.98?

Reply:
991 is sufficient I think. No need to overdo it

Reply:
We cannot use CSS variables as an argument to width of media queries until and unless we use SASS, etc.
https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries
Hence I believe, we have to keep it as it is with value of 991px.